src/main/resources/lang.yml

```diff
@@ -171,6 +178,7 @@ list-petmastergive: "Pets Given to Another Player"
 list-petmasterreceive: "Pets Received from Another Player"
 list-smelting: "Items Smelt"
 list-commands: "Other Achievements"
+list-customs: "Customs Achievements"
```

Comment:
For players, the difference between "Other Achievements" and "Customs Achievements" may be confusing. What do you think about "Other Achievements" and "Custom Categories" to make things a little clearer?